spec/truffle/launcher_spec.rb

```diff
@@ -56,58 +56,63 @@ def should_print_full_java_command(options, env: {})
     $?.success?.should == true
   end
 
-  it "prints the full java command with -J-cmd" do
-    should_print_full_java_command "-J-cmd --version"
-  end
+  unless Truffle.native?
```

Comment:
:+1: 
Could you use `guard_not` though? (details why in https://github.com/ruby/spec/commit/08434b263677766a7bf0bd1d39e763835029460e)

Reply:
Or rather `guard -> { !Truffle.native? } do`, which seems more commonly used used in spec/truffle.